lib/ohai/application.rb

```diff
@@ -34,8 +34,13 @@ class Ohai::Application
   option :directory,
     short: "-d DIRECTORY",
     long: "--directory DIRECTORY",
-    description: "A directory to add to the Ohai plugin search path",
-    proc: lambda { |path| Ohai::Config.platform_specific_path(path) }
+    description: "A directory to add to the Ohai plugin search path. If passing multiple directories comma separate directories.",
+    proc: lambda { |paths|
+      paths = paths.split(",")
```

Comment:
I think the last time I asked about this is lost in community slack now, but we've been using passing the option multiple times as the preferred UX over comma separated or other list parsing.

`--config-option` is one example of this: https://github.com/chef/chef/pull/5045
We switched knife ec2 to this behavior for security group IDs: https://github.com/chef/knife-ec2/pull/439
Also knife `--chef-tag` and `--aws-tag`: https://github.com/chef/knife-ec2/pull/520